packages/attention/stories/Attention.stories.tsx

```diff
@@ -196,22 +197,83 @@ export function Popover() {
         placement='bottom'
         targetEl={targetEl}
         isShowing={show}
+        id='popover-attention-example'
+      >
+        <ul className='bg-white w-full text-center'>
+          <li
+            className='p-2 px-40'
+          >
+            Hello
+          </li>
+          <li
+            className='p-2 px-40'
+          >
+            World
+          </li>
+        </ul>
+      </Attention>
+    </div>
+  </div>
+  )
+}
+
+export function PopoverIconAsTargetEl() {
+  const [show, setShow] = React.useState(false)
+  const containerRef = React.useRef<HTMLDivElement>(null)
+  const targetEl = React.useRef<any>(null);
+
+  React.useEffect(() => {
+    function onBlurHandler(e) {
+      if (containerRef.current && !containerRef.current.contains(e.target)) {
+        setShow(false)
+      }
+    }
+    document.addEventListener('mousedown', onBlurHandler)
+    return () => {
+      document.removeEventListener('mousedown', onBlurHandler)
+    }
+  })
+
+  return (
+  <div className='flex items-center justify-center'>
+    <div ref={containerRef}>
+      <Button
+        small
```

Comment:
Not sure if it's a bug or not, but the small button's vertical padding cuts the icon 🤔 
![Screenshot 2024-04-09 at 10 49 17](https://github.com/warp-ds/react/assets/41303231/ff347e1a-02cb-4959-bee2-61dd877094fa)
Maybe until we figure out what's wrong, you could remove the `small` prop? 😄 It looks good in the regular variant:
![Screenshot 2024-04-09 at 10 49 27](https://github.com/warp-ds/react/assets/41303231/6ca8f9a7-824d-4c85-a26c-96b37ee825f7)

Reply:
ah nice catch! I can remove the `small` prop!

Reply:
Hmm although, I don't get that the small button's vertical padding cuts the icon: 
<img width="145" alt="Skärmavbild 2024-04-09 kl  11 02 18" src="https://github.com/warp-ds/react/assets/144792260/c16a9a6c-0afe-460c-a211-f2557059c389">
🤔

Reply:
Updated in this commit: [70eb1e0](https://github.com/warp-ds/react/pull/215/commits/70eb1e0fb7344ce19cfc1c4970caa8ab0ff95cfd)